javalin/src/main/java/io/javalin/http/servlet/JavalinServletContext.kt

```diff
@@ -129,8 +116,9 @@ open class JavalinServletContext(
     private val method by javalinLazy { super.method() }
     override fun method(): HandlerType = method
 
-    override fun handlerType(): HandlerType = handlerType
-    override fun endpoint(): Endpoint? = endpoint
+    override fun endpoints(): List<Endpoint> = endpointStack.toList()
```

Comment:
I don't think we need a copy of this list if we'll make it readonly on type level.